README.rst

```diff
@@ -31,6 +31,7 @@ There are a few build arguments defined in the Dockerfile and it is possible to
 - PGOLDVERSIONS="9.5 9.6 10 11"
 - DEMO=false # set to true to build the smallest possible image which will work only on Kubernetes
 - TIMESCALEDB_APACHE_ONLY=true # set to false to build timescaledb community version (Timescale License)
+- ADDITIONAL_LOCALES= # comma separated locales to build into image (example: de_DE,pl_PL)
```

Comment:
I wonder if we should be using a comma as a separator, just space character (like it is used in the `PGOLDVERSIONS`) would work nice and will make `echo | sed` call unnecessary:
https://github.com/zalando/spilo/pull/513/files#diff-8918d403d0c48088cebb6ed78df4fecd856e211f7522e06ee423383b17bd8453R32
https://github.com/zalando/spilo/pull/513/files#diff-8918d403d0c48088cebb6ed78df4fecd856e211f7522e06ee423383b17bd8453R44

Reply:
Changed comma to space character